### ISSUE TITLE
Promote Wikipedia upgrade to Vega 3 / VL 2

### ISSUE THREAD

**Issue by nyurik** [vega/vega#1301]
Per discussion with @domoritz about promoting Wikipedia to start using Vega 3 & Vega-Lite 2.  See [Phabricator ticket](https://phabricator.wikimedia.org/T165118).  One option is to find a student to tackle this task, and possibly even apply for a Wikimedia grant to do this.

**Comment by jheer** [comment]
I'm super supportive of this, but I'm not sure what to do with it as a GitHub issue here. Is the goal to encourage someone to work on this? If so, you might add a "help-wanted" label along with more specific guidance on how to get involved. Also, it would be good to post on Vega Slack, Google Group, and also on Twitter (the vega_vis account can retweet)!

**Comment by nyurik** [comment]
correct, this is a tracking issue to attract a (student?) contributor to work on this, and to point them to the corresponding Phabricator ticket.  @jheer would you like to promote it too?  You might know better/additional channels for this.

**Comment by hjhimanshu01** [comment]
@nyurik @jheer , Hello Himanshu here , I have been contributing to WikiEduDashboard while applying for GSOC and got to know about VEGA from phabricator , loved this platform and would like to contibute to this issue in GSOC , could you help me to get started on this?

Link to PR's on wikiEduDashboard : https://github.com/WikiEducationFoundation/WikiEduDashboard/pulls/hjhimanshu01

**Comment by nyurik** [comment]
@hjhimanshu01 thanks! Your first steps would be to get mediawiki running on your machine.  The "new  and cool" way is to use [mw docker](https://github.com/addshore/mediawiki-docker-dev#readme), the old and somewhat slow (perf-wise) method is to use Vagrant or even to set up all the dependencies locally.  See also https://www.mediawiki.org/wiki/How_to_become_a_MediaWiki_hacker#Docker_development_environment

**Comment by hjhimanshu01** [comment]
@nyurik , Thanks for a warm welcome! :) , I just had an error , wikimedia gerrit link on docs isn't working , also tried other methods to reach that out so that it could be worked out using docker , is it possible to reach you out on IRC to discuss other details too?

**Comment by hjhimanshu01** [comment]
@domoritz, same here , it's hard to express , would love to contribute here , Let's get started then! :)

**Comment by nyurik** [comment]
@hjhimanshu01 i think Gerrit was down yesterday :)

**Comment by hjhimanshu01** [comment]
@nyurik , it's still down I think.

**Comment by nyurik** [comment]
@hjhimanshu01 which link to be sure?

**Comment by nyurik** [comment]
@hjhimanshu01 also, you can always talk to us on Slack at https://communityinviter.com/apps/vega-js/join

**Comment by hjhimanshu01** [comment]
@nyurik , links working right now , will discuss further on slack!

**Comment by ChenXin360104** [comment]
@nyurik Excuse me, I can find the #wiki channel on Slack. Which channel should I join to discuss?

**Comment by nyurik** [comment]
> @nyurik Excuse me, I cannot find a #wiki channel on Slack. Which channel should I join to discuss?

**#general** works :)

**Comment by ChenXin360104** [comment]
I found only selected users can post to #general 😰

Actually, what I'm wondering is:
In the [phab:T165118](https://phabricator.wikimedia.org/T165118), Milimetric said the Graphoid service and the Graph extension needed to be restructure, and it was posted on Feb. 5th. Are there any new progress since then? Where can I get the latest news?

I can't directly ask this question in Phabricator because my account is still not approved by the administrator. 😅

**Comment by hjhimanshu01** [comment]
@ChenXin360104 , contact them on Freenode or zulip , I think they are more active on that platform , but that depends on whom you are trying to contact.

**Comment by domoritz** [comment]
#general is locked. I created a #wiki channel.

**Comment by ChenXin360104** [comment]
I found my proposal wasn't accepted by GSoC2019, but I still want to join this project as a volunteer.
What can I do for you?

**Comment by domoritz** [comment]
@ChenXin360104 definitely! Can you join the slack and coordinate with @hjhimanshu01?

**Comment by ChenXin360104** [comment]
Sure. I already joined to the #wiki channel, and I will talk with @hjhimanshu01 in Slack.

**Comment by hydrosquall** [comment]
Just checking if this thread is still relevant to keep open - at this point, I think the goal might be to upgrade to VL 5!

**Comment by PBI-David** [comment]
@hydrosquall  - I think this can be closed. The orignal Phabicrator ticket states Apache eCharts was selected:

![Image](https://github.com/user-attachments/assets/ff7d6634-5840-4bcd-a45a-3e906045bffb)

**Comment by hydrosquall** [comment]
Ah thanks for investigating - that's too bad!
